torch/csrc/distributed/c10d/Ops.cpp

```diff
@@ -328,5 +350,35 @@ c10::intrusive_ptr<ProcessGroup::Work> barrier(
   return op.call(process_group, opts.device_ids, opts.timeout.count());
 }
 
+c10::intrusive_ptr<ProcessGroup::Work> send(
+    const c10::intrusive_ptr<ProcessGroup>& process_group,
+    at::TensorList tensors,
+    int64_t dstRank,
+    int64_t tag) {
+  static auto op = c10::Dispatcher::singleton()
+                       .findSchemaOrThrow("c10d::send", "")
```

Comment:
hmm would we not find the schema by any chance?

Reply:
Why would you have such concern? Do you happen to know another op called "c10d::send" as well?